src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -154,6 +156,28 @@ private void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
       logger.info("Version: Skipping version determination");
     }
 
+    if (config.server.testRunnerEnvironmentScript != null) {
+      TestRunnerEnvironmentScript ghContextScript =
+          config
+              .server
+              .testRunnerEnvironmentScript
+              .scriptClass
+              .getDeclaredConstructor()
+              .newInstance();
+      gitHubContextResult = ghContextScript.getTestRunnerEnvironmentContext();
+    } else {
+      try {
+        Class<?> scriptClassGeneric = GitHubActionsWorkflowRunContext.class;
+        Class<? extends TestRunnerEnvironmentScript> scriptClass =
+            (Class<? extends TestRunnerEnvironmentScript>) scriptClassGeneric;
+        gitHubContextResult =
+            scriptClass.getDeclaredConstructor().newInstance().getTestRunnerEnvironmentContext();
```

Comment:
If you're defaulting to a `GitHubActionsWorkflowRunContext` script, do you need all these Class generics or could you just do something like `new GitHubActionsWorkflowRunContext().getTestRunnerEnvironmentContext()`?

Reply:
I have refactored the code.